lib/cocoapods/command/lib/create.rb

```diff
@@ -25,12 +25,14 @@ def self.options
         def initialize(argv)
           @name = argv.shift_argument
           @template_url = argv.option('template-url', TEMPLATE_REPO)
+          @help = argv.flag?('help')
```

Comment:
I feel like we should be able to fix this in CLAide, since we are calling `super` before grabbing the `remainder!`?

Reply:
how about adding a method like `remainder_except_help!` ? I have try to fix this in CLAide, but do not find a proper way